script.js

```diff
@@ -1,4 +1,150 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var SCISSORS = "scissors";
+  var PAPER = "paper";
+  var STONE = "stone";
+  var reversedSCISSORS = "reversed scissors";
+  var reversedPAPER = "reversed paper";
+  var reversedSTONE = "reversed stone";
+  var CommputerObject = randomObjectByComputer();
+  var myOutputValue =
+    "Invalid input. Please enter 'scissors', 'paper', 'stone', 'reversed scissors', 'reversed paper' or 'reversed stone' to start the game.";
+
+  // scissors and scissors draw condition
+  if (
+    (input == SCISSORS && CommputerObject == SCISSORS) ||
+    (input == reversedSCISSORS && CommputerObject == SCISSORS)
+  ) {
+    console.log("computer choice");
+    console.log(CommputerObject);
```

Comment:
for lines 17-18, if your aim is to put a label (line 17) to the result of the function (line 18) you can refactor it to :
console.log("computer choice", CommputerObject)